pkg/odo/cli/service/create.go

```diff
@@ -383,6 +429,8 @@ func NewCmdServiceCreate(name, fullName string) *cobra.Command {
 		serviceCreateCmd.Example += fmt.Sprintf("\n\n") + fmt.Sprintf(createOperatorExample, fullName)
 		serviceCreateCmd.Flags().StringVar(&o.CustomResource, "crd", "", "The name of the CRD of the operator to be used to create the service")
 		serviceCreateCmd.Flags().BoolVar(&o.DryRun, "dry-run", false, "Print the yaml specificiation that will be used to create the service")
+		// remove this feature after enabling service create interactive mode for operator backed services
```

Comment:
why remove this feature? I mean why not keep this flag?

Reply:
@mik-dass our goal is to let users create a service interactively just as they used to with Service Catalog. And be able to provide parameters. But those two need more time and discussion so, in the interim, we're adding this feature to create a service from a yaml file.

We've documented that in the acceptance criteria on #2613 (` User should not be required to provide or write any CR definitions or YAML`).

Reply:
OK